setup-scripts/disk-usage-dump.sh

```diff
@@ -23,11 +23,21 @@ dump () {
     echo
     df -h /home/user
 
+    # Root dirs
+    echo
+    echo "Root dirs:"
+    docker run -v /:/host-root alpine:3.15.0 sh -c 'du -hs /host-root/*' | sed 's#/host-root##'
```

Comment:
I am not sure we need this part. Historically, this has never been a problem and it would add more pressure on the disk.

Reply:
On `us-ny1` we had full disk while `/user/home` took only 70GB this night.